iaso/api/org_unit_change_request_configurations/views.py

```diff
@@ -0,0 +1,106 @@
+import django_filters
+from django.db.models import Q
+from rest_framework import viewsets, filters, serializers
+from rest_framework.decorators import action
+from rest_framework.response import Response
+
+from iaso.api.org_unit_change_request_configurations.filters import OrgUnitChangeRequestConfigurationListFilter
+from iaso.api.org_unit_change_request_configurations.pagination import OrgUnitChangeRequestConfigurationPagination
+from iaso.api.org_unit_change_request_configurations.serializers import (
+    OrgUnitChangeRequestConfigurationListSerializer,
+    OrgUnitChangeRequestConfigurationRetrieveSerializer,
+    OrgUnitChangeRequestConfigurationWriteSerializer,
+    OrgUnitChangeRequestConfigurationUpdateSerializer,
+    OrgUnitTypeNestedSerializer,
+    OrgUnitChangeRequestConfigurationAuditLogger,
+    ProjectIdSerializer,
+)
+from iaso.models import OrgUnitChangeRequestConfiguration, OrgUnitType, Project
+
+
+class OrgUnitChangeRequestConfigurationViewSet(viewsets.ModelViewSet):
+    f"""OrgUnitChangeRequestConfiguration API
+
+    GET /api/orgunits/changes/configs
+    GET /api/orgunits/changes/configs/<id>
+    POST /api/orgunits/changes/configs/
+    PATCH /api/orgunits/changes/configs/<id>
+    DELETE /api/orgunits/changes/configs/<id>
+    GET /api/orgunits/changes/configs/check_availability/?project_id=<project_id>
+    """
+    filter_backends = [filters.OrderingFilter, django_filters.rest_framework.DjangoFilterBackend]
+    filterset_class = OrgUnitChangeRequestConfigurationListFilter
+    ordering_fields = [
+        "id",
+        "project__name",
+        "org_unit_type__name",
+        "org_units_editable",
+        "created_at",
+        "updated_at",
+        "created_by__username",
+        "updated_by__username",
+    ]
+    http_method_names = ["get", "post", "patch", "delete"]
+    pagination_class = OrgUnitChangeRequestConfigurationPagination
+
+    def get_queryset(self):
```

Comment:
I think we don't want to allow a user to see all the OUCRC of all the accounts from the API.
This should be filtered by the account of the user and project.account and maybe by user.iaso_profile.project ? Not sure about the second one ...